Package.swift

```diff
@@ -1,15 +1,38 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.3
 import PackageDescription
 
 let package = Package(
     name: "jaeger-client-swift",
     products: [
         .library(name: "Jaeger", targets: ["Jaeger"]),
+        .library(name: "ZipkinRecordingStrategy", targets: ["ZipkinRecordingStrategy"]),
```

Comment:
is this jaeger wording? or would recorder be ok?

This would be "exporter" I feel?

Reply:
JaegerZipkinExporter?

Reply:
> is this jaeger wording? or would recorder be ok?
> This would be "exporter" I feel?
> @ktoso

Good point 👍

I looked up the zipkin-compatibility in the official Java client. They use `ZipkinV2Reporter` for the JSON-based HTTP transport, and `ZipkinSender` for Thrift which supports different transports: https://github.com/jaegertracing/jaeger-client-java/tree/dfb9d8b9dc849f2e6f880b44b26fb77dfc894a02/jaeger-zipkin#zipkin-2-reporters

What do you think about naming the library `ZipkinReporting` and the reporter named `ZipkinReporter`?

Reply:
Heh i guess depends whose naming you want to follow, as we try to be otel inspired, otel wording is "exporter"

but `reporter` is okey too if you want; "recorder" is definitely wrong though.

`ZipkinReporting` or `ZipkinFormat` could be fine yea

Reply:
> Heh i guess depends whose naming you want to follow, as we try to be otel inspired, otel wording is "exporter"
> @ktoso

I think in this case it would make sense to align with existing official Jaeger clients.

Reply:
Yeah makes sense -- so it seems it'd be reporter / reporting, LGTM